src/backend/src/utils/change-requests.utils.ts

```diff
@@ -182,3 +182,17 @@ export const validateProposedChangesFields = async (
     }
   }
 };
+
+/**
+ * Determines if there are other open unreviewed change requests for this wbs element
+ * @param wbsElemId the wbs element id to find CRs with
+ * @throws if the WBS element has open unreviewed change requests
+ *
+ */
+export const validateUnreviewedOpenCRs = async (wbsElemId: number) => {
+  const openCRs = await prisma.change_Request.findMany({
+    where: { wbsElementId: wbsElemId, accepted: false }
```

Comment:
oh I see this is so we don't have merge conflicts ever. Add a comment to where this is called to specify that's why we need to enforce this invariant, I think this should be looking for any change request where dateReviewed is undefined instead. We don't want this error to throw if there are any denied change requests on the same wbs element